classes/Event/class.xoctEventGUI.php

```diff
@@ -557,9 +566,13 @@ public function streamVideo() {
             }
 		}, $medias);
 
-		$segments = array_filter($publication->getAttachments(), function (xoctAttachment $attachment) {
-			return (in_array("segments", $attachment->getTags()));
+		$segmentFlavor = xoctPublicationUsage::find(xoctPublicationUsage::USAGE_SEGMENTS)->getFlavor();
+
+
+		$segments = array_filter($publication->getAttachments(), function (xoctAttachment $attachment) use ( &$segmentFlavor)  {
+			return strpos($attachment->getFlavor(), $segmentFlavor) !== FALSE;
```

Comment:
@rfharokh ich habe hier die Überprüfung auf Tags entfernt. Den neuen Konfigurationspunkt braucht es ja demnach nicht mehr und wurde daher auch entfernt.

Reply:
@theodortruffer Ich hab es jetzt aus unserem Fork auch entfernt. Wir werden das nächste Woche noch alles mit Opencast 6.2 testen. Bis jetzt kann ich keinen Fehler feststellen.

Reply:
@rfharokh Wunderbar, ich werde dies dann diese oder nächste Woche in unser Repo mergen.

Auf diesem Branch hier wäre nun zusätzlich die neuste Version des Paella-Players via npm eingebunden. Allerdings scheint's noch nicht ganz zu klappen, zumindest mit dem Opencast-Server der Uni Köln. Könnten Sie dies bei Gelegenheit kurz bei Ihnen testen?